RUFAS/general_constants.py

```diff
@@ -27,6 +27,14 @@ class GeneralConstants:
         PERCENTAGE_TO_FRACTION: The conversion factor from a percentage to a fraction.
         FRACTION_TO_PERCENTAGE: The conversion factor from a fraction to a percentage.
 
+        EMBEDDED_NITROGEN_FERTILIZER_EMISSIONS_FACTOR:
+         Constants for calculating the embedded emissions of synthetic nitrogen. Reference IPCC 2021.
+        EMBEDDED_PHOSPHORUS_FERTILIZER_EMISSIONS_FACTOR:
+         Constants for calculating the embedded emissions of synthetic PHOSPHORUS. Reference GWP 100.
+        EMBEDDED_POTASSIUM_FERTILIZER_EMISSIONS_FACTOR:
+         Constants for calculating the embedded emissions of synthetic POTASSIUM. Reference BASF's Eco-efficiency
+         analysis tool.
```

Comment:
These constants need units. Also, there is nothing wrong with putting them in the general constants file but my two cents is that this isn't necessary, because these are not general constants - they are very specific to the emissions module.
```suggestion
        EMBEDDED_NITROGEN_FERTILIZER_EMISSIONS_FACTOR : float
            Constants for calculating the embedded emissions of synthetic nitrogen. Reference IPCC 2021.
        EMBEDDED_PHOSPHORUS_FERTILIZER_EMISSIONS_FACTOR : float
            Constants for calculating the embedded emissions of synthetic phosphorus. Reference GWP 100.
        EMBEDDED_POTASSIUM_FERTILIZER_EMISSIONS_FACTOR : float
            Constants for calculating the embedded emissions of synthetic potassium. Reference BASF's Eco-efficiency
            analysis tool.
```

Reply:
I do have this debate in mind, thanks for your suggestions and I do think it makes sense.